static/hooks/usePolling.jsx

```diff
@@ -30,19 +30,11 @@ function usePolling({url, handleRecieveData, pollingActive = true}) {
   const pollTimeoutRef = useRef(undefined);
 
   /**
-   * Cancels pending polling timeouts
-   */
-  const cancelPolling = useCallback(() => {
-    window.clearTimeout(pollTimeoutRef.current);
-    pollTimeoutRef.current = undefined;
-  }, []);
-
-  /**
-   * Beings the polling cycle
+   * Starts chain triggering the makeRequest
    */
   const triggerPolling = useCallback(async () => {
     if (!pollingActive) {
-      return cancelPolling;
```

Comment:
This was trying to do what this PR does, but it was returning a Promise since triggerPolling is async, so even without the `void` before triggerPolling, useEffect would still be unhappy.